src/main.zig

```diff
@@ -2193,6 +2193,7 @@ pub fn cmdBuild(gpa: *Allocator, arena: *Allocator, args: []const []const u8) !v
             }
             // Search up parent directories until we find build.zig.
             var dirname: []const u8 = cwd_path;
+            var prevDirname: []const u8 = dirname;
```

Comment:
Please use snake case for variables. Also the type can be inferred. 
```suggestion
            var prev_dirname = dirname;
```

Reply:
Made the suggested changes.